modulefiles/cheyenne.gnu/fv3

```diff
@@ -37,3 +37,11 @@ module load NCEPlibs/9.9.9
 ## SIONlib library
 ##
 module load SIONlib/1.7.4
+
+##
+## set cmake compielrs
+##
```

Comment:
compilers

Reply:
Thanks, fixed it in both modulefiles/cheyenne.gnu/fv3 and modulefiles/cheyenne.intel/fv3.